platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/generic/GenericPageRenderer.java

```diff
@@ -156,7 +153,7 @@ protected ReportRenderer<TestTreeModel, SimpleHtmlWriter> getContentRenderer() {
                     }
                 }
                 if (!info.getMetadatas().isEmpty()) {
-                    perRootInfoTabsRenderer.add("metadata", new PerRootTabRenderer.ForMetadata(rootIndex, perRootInfoIndex, metadataRendererRegistry));
+                    perRootInfoTabsRenderer.add("metadata", new PerRootTabRenderer.ForMetadata(rootIndex, perRootInfoIndex));
```

Comment:
❔ Didn't we want to change the name of the tab, too?  Is that worth doing in this PR, too?